caravel/assets/spec/javascripts/sqllab/QuerySearch_spec.jsx

```diff
@@ -15,7 +15,7 @@ describe('QuerySearch', () => {
 
   it('should have two Select', () => {
     const wrapper = shallow(<QuerySearch />);
-    expect(wrapper.find(Select)).to.have.length(2);
+    expect(wrapper.find(Select)).to.have.length(4);
```

Comment:
more like tests that test the logic rather than presence of the components.
it you select option expect that state is updated, etc
if you execute search - except url is like expected